pkg/ext/apiserver.go

```diff
@@ -239,20 +239,22 @@ func InstallStore[T runtime.Object, TList runtime.Object](
 		apiGroup.VersionedResourcesStorageMap[gvk.Version] = make(map[string]rest.Storage)
 	}
 
-	delegate := &delegate[T, TList]{
-		scheme: s.scheme,
-
-		t:            t,
-		tList:        tList,
-		singularName: singularName,
-		gvk:          gvk,
-		gvr: schema.GroupVersionResource{
-			Group:    gvk.Group,
-			Version:  gvk.Version,
-			Resource: resourceName,
+	delegate := &delegateError[T, TList]{
```

Comment:
Please don't use variable names that shadow outer names (here `delegate` is already the name of a local type).

Reply:
Ok, just renamed it. Thanks!